configs/DRONEERH743DUAL/config.h

```diff
@@ -0,0 +1,182 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU                   STM32H743
+#define BOARD_NAME                      DRONEERH743DUAL
+#define MANUFACTURER_ID                 DRNR
+
+// Miscellaneous
+#define LED0_PIN                        PC4  //ADC12_INP4
+#define LED1_PIN                        PC5  //ADC12_INN4/ADC12_INP8
+#define LED_STRIP_PIN                   PA0  //TIM2_CH1/TIM5_CH1/UART4_TX/ADC1_INP16
+#define BEEPER_PIN                      PD14 //TIM4_CH3
+#define USB_DETECT_PIN                  PA10 //TIM1_CH3/USART1_RX
+
+// Camera control
+#define CAMERA_CONTROL_PIN              PA1  //TIM2_CH2/TIM5_CH2/TIM15_CH1N/UART4_RX/ADC12_INP1
+
+// PINIO1 -- VTX Switch
+#define PINIO1_PIN                      PA2  //TIM2_CH3/TIM5_CH3/TIM15_CH1/USART2_TX/ADC12_INP14
+#define PINIO1_CONFIG                   1
```

Comment:
This should probably be inverted so the default is on if the user does not configure this mode. 
```suggestion
#define PINIO1_CONFIG                   129
```